boa3_test/tests/boatestcase.py

```diff
@@ -527,6 +527,21 @@ async def get_latest_block(cls) -> block.Block:
             best_block_hash = await rpc_client.get_best_block_hash()
             return await rpc_client.get_block(best_block_hash)
 
+    @classmethod
+    async def get_exec_fee_factor(cls) -> int:
+        async with noderpc.NeoRpcClient(cls.node.facade.rpc_host):
+            return await cls.node.facade.test_invoke(PolicyContract().exec_fee_factor())
+
+    @classmethod
+    async def get_fee_per_byte(cls) -> int:
+        async with noderpc.NeoRpcClient(cls.node.facade.rpc_host):
+            return await cls.node.facade.test_invoke(PolicyContract().fee_per_byte())
+
+    @classmethod
+    async def get_storage_price(cls) -> int:
+        async with noderpc.NeoRpcClient(cls.node.facade.rpc_host):
+            return await cls.node.facade.test_invoke(PolicyContract().storage_price())
+
```

Comment:
since these are only used in `test_policy` it doesn't need to be here, you can move these methods to there